docs/data/index.rst

```diff
@@ -123,22 +128,45 @@ Filtering data
 
 Sherpa supports filtering data sets; that is, temporarily removing
 parts of the data (perhaps because there are problems, or to help
-restrict parameter values).
+restrict parameter values). There are routines to filter the data
+and to find out what bins have been selected.
 
+The :py:meth:`~sherpa.data.Data.ignore` and
+:py:meth:`~sherpa.data.Data.notice` methods are used to
+define the ranges to exclude or include.
 The :py:attr:`~sherpa.data.Data.mask` attribute indicates
 whether a filter has been applied: if it returns ``True`` then
-no filter is set, otherwise it is a bool array
+no filter is set, ``False`` when all data has been filtered out,
+otherwise it is a bool array
 where ``False`` values indicate those elements that are to be
-ignored. The :py:meth:`~sherpa.data.Data.ignore` and
-:py:meth:`~sherpa.data.Data.notice` methods are used to
-define the ranges to exclude or include. For example, the following
+ignored. For example, the following
 hides those values where the independent axis values are between
 21.2 and 22.8::
 
     >>> d1.ignore(21.2, 22.8)
     >>> d1.x[np.invert(d1.mask)]
     array([21.5, 22. , 22.5])
 
+.. note::
+   The meaning of the range selected by the `notice` and `ignore` calls
+   depends on the data class: :py:class:`sherpa.data.Data1DInt` and
+   :py:class:`sherpa.astro.data.DataPHA` classes can treat the
+   upper limit differently (it in inclusive or exclusive)
+   and 2D datasets such as :py:class:`sherpa.data.Data2D` can
+   use filters that use both axes.
+
+The :py:meth:`~sherpa.data.Data1D.get_filter` method will report a
+string representation of the existing filter, so here showing that
+the bins between 21 and 23 have been ignored::
+
+    >>> print(d1.get_fiter())
+    20.0000:21.0000,23.0000:39.5000
+
+.. note::
+   The filter does not record the requested changes - that is here
+   the 21.2 to 21.8 arguments to :py:meth:`~sherpa.data.Data.ignore` -
```

Comment:
Do you mean 22.8 from line 146 rather than 21.8?

```suggestion
   the 21.2 to 23.8 arguments to :py:meth:`~sherpa.data.Data.ignore` -
```

Reply:
It should be 22.8, as you say, and not 21.8 or 23.8 :-)

Reply:
I'll update this later as we'll need a rebase for #1216

Reply:
Fixed in https://github.com/sherpa/sherpa/pull/1219/commits/c84975ba4ffd11b065f01f8bbe63de2c0285f3b0

Reply:
Which is now https://github.com/sherpa/sherpa/pull/1219/commits/c70b0f02c23a77de7241e704e86769c60c63a9f0